src/pages/Broker/Broker.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import ImmutablePropTypes from 'react-immutable-proptypes';
+import { connect } from 'react-redux';
+import { Row, Col } from 'react-bootstrap';
+import Page from '../../components/layout/Page';
+import {
+  fetchBrokerStats,
+  freezeBroker,
+  unfreezeBroker
+} from '../../redux/modules/broker';
+import { brokerStatsSelector } from '../../redux/selectors/broker';
+import { FormattedMessage } from 'react-intl';
+import BrokerButtons from '../../components/Broker/BrokerButtons/BrokerButtons';
+import StatsList from '../../components/Broker/StatsList/StatsList';
+
+class Broker extends Component {
+  static loadAsync = (params, dispatch) =>
+    Promise.all([dispatch(fetchBrokerStats())]);
+
+  componentWillMount = () => this.props.loadAsync();
+
+  render() {
+    const {
+      stats,
+      refreshBrokerStats,
+      freezeBroker,
+      unfreezeBroker
+    } = this.props;
+    return (
+      <Page
+        title={
+          <FormattedMessage
+            id="app.broker.title"
+            defaultMessage="Broker Management"
+          />
+        }
+        description={
+          <FormattedMessage
+            id="app.broker.description"
+            defaultMessage="Management of broker backend service"
+          />
+        }
+        resource={stats}
+      >
+        {stats =>
+          <div>
+            <BrokerButtons
+              refreshBrokerStats={refreshBrokerStats}
+              freezeBroker={freezeBroker}
+              unfreezeBroker={unfreezeBroker}
+            />
+            <Row>
+              <Col lg={6}>
+                <StatsList stats={stats} />
```

Comment:
Not the buttons nor the stats list have any permission checks. Even if we have no permission hints here, the contents should be at least hidden for users who do not posses the super-admin role.

Reply:
This should be solved on router level

Reply:
Maybe it should, but it is not (yet). And we have no way how to implement it simply into the router.

Reply:
Btw. loadAsync should test the permissions as well to prevent running a stats request to API which is doomed anyway.